lib/routes/account.js

```diff
@@ -1125,11 +1147,23 @@ module.exports = function (
                * 3) Verify account email if not already verified.
                */
               return db.verifyTokens(code, account)
+                .then(function () {
+                  log.info({
+                    op: 'account.signin.confirm.success',
+                    uid: account.uid.toString('hex'),
+                    code: request.payload.code
+                  })
+                })
                 .catch(function (err) {
                   if (err.errno === error.ERRNO.INVALID_VERIFICATION_CODE && butil.buffersAreEqual(code, account.emailCode)) {
                     // The code is just for the account, not for any sessions
                     return true
                   }
+                  log.error({
+                    op: 'account.signin.confirm.error',
```

Comment:
I think we should give this one a different name, e.g. `account.signin.confirm.invalid`, so that we can easily tease apart all the different cases when analyzing the metrics.